swim/member.go

```diff
@@ -73,6 +90,71 @@ func (m Member) checksumString(b *bytes.Buffer) {
 	b.WriteString(m.Address)
 	b.WriteString(m.Status)
 	b.WriteString(strconv.FormatInt(m.Incarnation, 10))
+	m.Labels.checksumString(b)
+}
+
+// checksumString adds the label portion of the checksum to the buffer that is
+// passed in. The string will not be appended in the case where labels are not
+// set on this member. This is for backwards compatibility reasons with older
+// versions.
+func (l LabelMap) checksumString(b *bytes.Buffer) {
```

Comment:
I think it is quite closely related to the member at this moment, however with the thoughts @mennopruijssers had earlier this week (to modularize checksums and potentially keep multiple) it would make sense to put the logic of the actual membership checksum, which includes the label checksumming, to its own file.

With regards to this function, now that the checksum calculation (the actual number) of the labels is extracted and reused below the body of this function might as well be collapsed into the `Member#checksumString` function since it is very much related with how a member is checksummed.